ci/travis/determine_tests_to_run.py

```diff
@@ -0,0 +1,93 @@
+# Script used for checking changes for incremental testing cases
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import subprocess
+
+
+def list_changed_files(commit_range):
+    """
+    Returns the list of files changed within commit_range.
```

Comment:
Format as `"""Returns the list of files changed within commit_range."""`, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for the style